XiEditor/EditViewController.swift

```diff
@@ -148,11 +150,15 @@ class EditViewController: NSViewController, EditViewDataSource, FindDelegate, Sc
 
                 statusBar.updateStatusBarColor(newBackgroundColor: self.theme.background, newTextColor: self.theme.foreground, newUnifiedTitlebar: unifiedTitlebar)
                 findViewController.updateColor(newBackgroundColor: self.theme.background, unifiedTitlebar: unifiedTitlebar)
-
-                if color.isDark && unifiedTitlebar {
-                    window.appearance = NSAppearance(named: NSAppearance.Name.vibrantDark)
+                
+                if #available(OSX 10.14, *) {
+                    // Let system decide, disregarding active theme color
```

Comment:
When `unifiedTitlebar == true`, we should be using the theme color instead of the system appearance here.